Add some docs for the extenders and export/import (#302)
kubernetes-sigs/kube-scheduler-simulator: simulator/docs/extender.md

> ## Extender

Reviewer: ```suggestion
## Integrate your scheduler extender into the simulator
```

> This document describes how to use your `Extenders` in the scheduler running in the simulator.

Reviewer: ```suggestion
This document describes how to use your scheduler extenders in the scheduler running in the simulator.
```

> The `Extender` is one of the features of the scheduler's webhook on kubernetes.

Reviewer: ```suggestion
The scheduler extender is the webhook-based scheduler extension.
```

> The simulator stores the results of each Extender in the annotation of a pod.

Reviewer: ```suggestion
The simulator adds the results of each extender to Pod's annotation like it does for the scheduler plugins.
```

(Note that it's not related to the [`plugin-extender`](./plugin-extender.md) which is one of the our simulator's feature. 
(Sorry for the confusing name 😅))

Note: This function uses api-server on the simulator to store the results.
Therefore, **if the scheduler is not connected to the simulator's api-server,

> this feature is not available.**

Reviewer: I don't get it. Can you elaborate?

Author: Oops, `api-server`→`HTTP server on the simualtor`.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/302/files#diff-512bab2a023bf2e92a0bfdf2995072f49029ae70faa536b01161fa221de09201R11-R13

Author: ```md
Note: This feature is not available in [external scheduler](./external-scheduler.md).
```

## How to use

> Set the path of the scheduler configuration file configured the Extender to be used,
> to the `kubeSchedulerConfigPath` value in the Simulator Server Configuration.

Reviewer: ```suggestion
You need to configure your extender in KubeSchedulerConfig.
(via [the simulator config](./simulator-server-config.md) or web ui)
```

> (No required special configuration is for the simulator to use this feature.)

Reviewer: seems no need

```suggestion
```

> For example, if you run the server on `http://localhost:8080/scheduler/`,

Reviewer: ```suggestion
For example, if your extender listens `http://localhost:8080/scheduler/`,
```
